packages/wallets/src/features/accounts/modules/ManualService/utils/utils.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import ClearPhoto from '../../../../../public/images/accounts/clear-photo.svg';
+import ClockIcon from '../../../../../public/images/accounts/clock-icon.svg';
+import DrivingLicenseIcon from '../../../../../public/images/accounts/driving-license.svg';
+import IdentityCardIcon from '../../../../../public/images/accounts/identity-card.svg';
+import ImageIcon from '../../../../../public/images/accounts/image-icon.svg';
+import LessThanEightIcon from '../../../../../public/images/accounts/less-than-eight-icon.svg';
+import NIMCSlipIcon from '../../../../../public/images/accounts/nimc-slip.svg';
+import PassportIcon from '../../../../../public/images/accounts/passport.svg';
```

Comment:
Can you double check whether we have this icons within `deriv/quill-icons`

Reply:
Hi @aizad-deriv,
We have some of it. Do have to request for others.